src/_extensions/publishing-guidelines.md

```diff
@@ -144,41 +144,7 @@ The final page gives you a chance to review all your choices before submitting y
 
 ## 4. Your extension is reviewed {#reviewed}
 
-When you submit your extension, we will evaluate it according to a set of acceptance criteria. If it doesn't satisfy all of them, the extension will be rejected. You can track the status of your submitted extensions at the [Submitted extensions](https://addons.opera.com/developer/) page.
-
-### Acceptance criteria {#acceptance-criteria}
```

Comment:
At line 24 there is a link to this header. That link needs to be removed as well.